Source/CarthageKit/BinaryJSONError.swift

```diff
@@ -21,10 +20,13 @@ extension BinaryJSONError: CustomStringConvertible {
 		switch self {
 		case let .invalidJSON(error):
 			return "invalid JSON: \(error)"
+
```

Comment:
My 2 cents: I find that the lack of whitespace in Carthage source code makes it very hard to read. I lean for more empty lines where it make sense.